src/utils/amplitude.ts

```diff
@@ -54,12 +55,21 @@ type OppfolgingsgrunnSendt = {
   };
 };
 
+type OptionSelected = {
+  type: EventType.OptionSelected;
+  data: {
+    select: string;
+    option: string;
+  };
+};
```

Comment:
Er det greit å fyre på `url` og her? 🤔 Er det tenkt at man skal legge ved label til select feltet på `select`? I så fall så kan vi jo kalle det `label`? 
```suggestion😊
type OptionSelected = {
  type: EventType.OptionSelected;
  data: {
    select: string;
    option: string;
    url: string;
  };
};
```

Reply:
Ja, kan godt legge til url. Tanken med select var å ha en beskrivelse av hvilken select det er. Label kunne funket, men i forhåndsvarsel-tilfellet så gir det kanskje ikke nok kontekst. Vil også ha med at det er i forhåndsvarsel-skjemaet som heller ikke urlen sier noe om :thinking:

Reply:
Et alternativ er å bruke`tekst`. Det er ofte brukt i NAV sin taksonomi for slike felter. Da er i tillegg dette gjenkjennbart mellom dette eventet, `ButtonClick` og `AccordionOpened` 😊

Reply:
Ja, det er bedre. Fikser!